auth-web/src/views/auth/HomeViewDev.vue

```diff
@@ -247,11 +247,16 @@ export default class HomeViewDev extends Vue {
     }
   }
 
-  @media only screen and (max-width: 600px) {
+  @media only screen and (max-width: 48rem) {
```

Comment:
Actually in THIS case it's better as px values (since those are easy to map to known screen sizes).

Just a note that 600px is a weird breakpoint. Usually it's something like 630 or 768 or 800. Don't change it, just sayin'.

Reply:
Update line 255 as well pls.

Reply:
K, will do!

Reply:
Resolved!